spec/models/content_asset_spec.rb

```diff
@@ -1,5 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe ContentAsset, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "with an attached file" do
+    before(:each) do
+      @content_asset = FactoryBot.create(:content_asset)
+    end
+
+    it "has a file attached" do
```

Comment:
RSpec supports a one-liner syntax for setting an expectation on the `subject`.
Using this, the test could be simplified to something along the lines of

```suggestion
 describe "#avatar" do
   subject { described_class.new(FILE_TO_ATTACH) }
   
   it { should be_attached }
 end
```

Reply:
i think i prefer the original syntax in this case